spec/models/current_claim_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe CurrentClaim, type: :model do
+  context "Two claims - ECP and LUP" do
+    let(:ecp_policy) { EarlyCareerPayments }
+    let(:lup_policy) { LevellingUpPremiumPayments }
+    let(:ecp_claim) { build(:claim, academic_year: "2022/2023", policy: ecp_policy) }
+    let(:lup_claim) { build(:claim, academic_year: "2022/2023", policy: lup_policy) }
+    let(:school) { create(:school) }
+
+    describe "#attributes=" do
+      it "sets the attributes on both claims" do
+        cc = CurrentClaim.new(claims: [ecp_claim, lup_claim])
+        cc.attributes = {"eligibility_attributes" => {"current_school_id" => school.id}}
+
+        expect(cc.claims.first.school.id).to eq(school.id)
+        expect(cc.claims.last.school.id).to eq(school.id)
```

Comment:
For tests like these I usually like to be more explicit, making sure it does the right thing:

```ruby
expect { cc.attributes = {"eligibility_attributes" => {"current_school_id" => school.id}} }
  .to change { cc.claims.first.school&.id }.from(nil).to(school.id)
  .and change { cc.claims.last.school&.id }.from(nil).to(school.id)
```

I think `CurrentClaim.new(claims: [ecp_claim, lup_claim])` and `{"eligibility_attributes" => {"current_school_id" => school.id}}` could also be memoized with a `let`.